.buildkite/benchmarks.pipeline.yml

```diff
@@ -0,0 +1,143 @@
+    - "LANG=C.UTF-8"
+    - "CARGO_TARGET_DIR=/var/tmp/artifacts"
+    - "CARGO_INSTALL_ROOT=/root/.cargo"
+    - "GOPROXY=https://proxy.golang.org/"
+    - "SLACK_WEBHOOK_URL"
+    - "METRICS_PUSH_ADDR"
+    - "METRICS_QUERY_ADDR"
+    - "METRICS_SOURCE_GIT_BRANCH"
+    - "METRICS_TARGET_GIT_BRANCH"
+    - "METRICS_LABELS_INSTANCE=$BUILDKITE_PIPELINE_NAME-$BUILDKITE_BUILD_NUMBER"
+    - "TESTS"
+  propagate-environment: true
+  unconfined: true
+
+docker_plugin: &docker_plugin
+  oasislabs/docker#v3.0.1-oasis1:
+    <<: *docker_plugin_default_config
+
+steps:
+  ###############################################################
+  # The following three steps are copied from code.pipeline.yml #
+  ###############################################################
+  ############
+  # Build jobs
+  ############
+  # This label needs to be synced with runtime-ethereum's
+  # .buildkite/scripts/download_utils.sh.
+  - label: Build Go node
+    command:
+      - .buildkite/go/build.sh
+
+      # Upload the built artifacts.
+      - cd /workdir/go/oasis-node
+      - buildkite-agent artifact upload oasis-node
+      - cd /workdir/go/oasis-node/integrationrunner
+      - buildkite-agent artifact upload integrationrunner.test
+      - cd /workdir/go/oasis-test-runner
+      - buildkite-agent artifact upload oasis-test-runner
+      - cd /workdir/go/oasis-net-runner
+      - buildkite-agent artifact upload oasis-net-runner
+      - cd /workdir/go/oasis-remote-signer
+      - buildkite-agent artifact upload oasis-remote-signer
+    plugins:
+      <<: *docker_plugin
+
+  # This label needs to be synced with runtime-ethereum's
+  # .buildkite/scripts/download_utils.sh.
+  - label: Build Rust runtime loader
+    command:
+      - .buildkite/rust/build_generic.sh /workdir -p oasis-core-runtime-loader
+      - .buildkite/rust/build_generic.sh /workdir -p test-long-term-client
+      - .buildkite/rust/build_generic.sh /workdir -p simple-keyvalue-client
+      - .buildkite/rust/build_generic.sh /workdir -p simple-keyvalue-enc-client
+      - .buildkite/rust/build_generic.sh /workdir -p simple-keyvalue-ops-client
+
+      # Upload the built artifacts.
+      - cd /var/tmp/artifacts/default/debug
+      - buildkite-agent artifact upload oasis-core-runtime-loader
+      # Clients for E2E tests.
+      - buildkite-agent artifact upload test-long-term-client
+      - buildkite-agent artifact upload simple-keyvalue-client
+      - buildkite-agent artifact upload simple-keyvalue-enc-client
+      - buildkite-agent artifact upload simple-keyvalue-ops-client
+    agents:
+      buildkite_agent_size: large
+    plugins:
+      <<: *docker_plugin
+
+  ####################
+  # Runtime build jobs
+  ####################
+  # This label needs to be synced with runtime-ethereum's
+  # .buildkite/rust/test_runtime_and_gateway.sh and .buildkite/scripts/download_utils.sh.
+  - label: Build key manager runtime
+    command:
+      - .buildkite/rust/build_runtime.sh keymanager-runtime
```

Comment:
This has changed in master, re-check and fix, also consider re-testing the pipeline before merging.